### PR TITLE
Add Paay MPI

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

**Comment by vova07** [comment]
please review

**Comment by vova07** [comment]
please review

File: spec/definitions/Mpi/NMI/NMIMpis.yaml [hunk 1]
```diff
@@ -8,4 +8,5 @@ properties:
     allOf:
       - $ref: "#/definitions/MpiName"
     enum:
+     - PaayMpi
```
Comment (slavcodev) [inline spec/definitions/Mpi/NMI/NMIMpis.yaml]: How does this work? I see another enum in `MpiName`, does this enum override from MpiName or supplements it?

Comment (vova07) [reply spec/definitions/Mpi/NMI/NMIMpis.yaml]: The enum is override!

**Pull request merged**